### ISSUE TITLE
War of the World

### ISSUE THREAD

**Issue by TwinNumberOne** [siggame/MegaMinerAI-Game-Ideas#69]
**TLDR**
-
- This is for mega miner
- You can mine asteroids or the planet
- Your goal is to destroy your enemy or to be the richest at the end of the time
- Asteroids, and space vehicles orbit above the planet
- You need to balance power, fuel, material

**Story**
-
The world has finally collapsed into two world governments and after a long cold war building up space research war has broken out each side controls half the planet but it is now or never to take out your the enemy and control the world.

**The Game**
-
**Victory**
Destroy the enemy or have the most resources by the end of the game (buildings and unites included)

**Map**
- There are two capitals on each side of the planet that each have allot of hp where you need multiple asteroids hitting it to take it out (each city good 1/8th of the planet)
- Asteroids orbit the planet, with different parts of the planet being lit by the sun at different times (the moving sun is optional I know it could be hard to program)
- Asteroids can have there speed reduce to make them crash into the planet using special devices
- asteroid speed does not change while being mined, and when fully mined the asteroid vanishes.
- The buildings have hp that when reduced to 0 destroys the building

**Movement**
Unites can be launched into orbit (basic orbit they need not do rocket science) where they can change their orbital speed to intercept different asteroids or move along the ground. On the ground it costs fuel to move and in space, it costs fuel to change your speed with a smaller base fuel cost. 

**Resource Collection**
There are two ways resources can be collected one is my upgrading the player's mines or by building unites to mine asteroids where you get both materials and some fuel from asteroids.
For the ground mines, fuel production should scale slower than material production as which switches at later tiers promoting mining In space all costing energy to upkeep. Meanwhile, energy should be a flat increase at each level but the costs should start making it cost to fuel or material to upkeep. 

**Unites**
Unites cost materials and energy to build (material/energy) I am going to be putting numbers to show the degree of materials or energy I think they should cost ranging from 0-5, with the same ranking being used for the other stats. Note each unit (0 means no cost)

**Infantry (Ground)**
The infantry is the most basic ground unit meant to be spammed out in large quantities. They can set up in a spot to establish a AA turret to shoot down Artillery rounds but it takes resources to do so (3).
| **Cost** | **Upkeep** | **HP** | **Movement** | **Attack** | **Fuel Cost** | **Range** |
|----|----|----|----|----|----|----|
|1/1|1/0|1|1|1|1|1|

**Transport (Ground)**
This unite is used to move fuel or infantry around the ground level.
| **Cost** | **Upkeep** | **HP** | **Movement** | **Attack** | **Fuel Cost** | **Range** |
|----|----|----|----|----|----|----|
|3/1|1/1|1|5|1|1|0|

**Artillery (Ground)**
Used as a powerful ranged attacker but takes time to move into position.
| **Cost** | **Upkeep** | **HP** | **Movement** | **Attack** | **Fuel Cost** | **Range** |
|----|----|----|----|----|----|----|
|3/3|3/1|2|1|5|3|3|

**Tank (Ground)**
The tank of the ground troops very powerful hard to kill
| **Cost** | **Upkeep** | **HP** | **Movement** | **Attack** | **Fuel Cost** | **Range** |
|----|----|----|----|----|----|----|
|5/5|5/3|5|3|5|5|2|

**Space Station (Space)**
This station is able to hold mined materials and fuel
| **Cost** | **Upkeep** | **HP** | **Movement** | **Attack** | **Fuel Cost** | **Range** |
|----|----|----|----|----|----|----|
|5/5|2/3|1|3|1|2|0|

**Shuttle (Space/Ground)**
It is the transport ship that is able to move from ground to space to put materials or fuel into the main stockpile
| **Cost** | **Upkeep** | **HP** | **Movement** | **Attack** | **Fuel Cost** | **Range** |
|----|----|----|----|----|----|----|
|3/1|2/1|1|5|1|3|0|

**Miner (Space)**
This ship can mine asteroids and has a limited cargo hold that it can unload unto transports and space stations
| **Cost** | **Upkeep** | **HP** | **Movement** | **Attack** | **Fuel Cost** | **Range** |
|----|----|----|----|----|----|----|
|5/5|2/3|1|3|1|3|1|

**Missle (Space/Ground)**
This unite can explode damaging or killing unites in a radius around it
| **Cost** | **Upkeep** | **HP** | **Movement** | **Attack** | **Fuel Cost** | **Range** |
|----|----|----|----|----|----|----|
|5/5|1/1|1|5|5|5|0|

**Steller Engine (Space)**
It can attack to a meteor to change its orbital speed
| **Cost** | **Upkeep** | **HP** | **Movement** | **Attack** | **Fuel Cost** | **Range** |
|----|----|----|----|----|----|----|
|5/5|1/5|1|1|1|5|0|

**AA platform (Space)**
It will shoot down missiles
| **Cost** | **Upkeep** | **HP** | **Movement** | **Attack** | **Fuel Cost** | **Range** |
|----|----|----|----|----|----|----|
|5/5|1/5|1|1|1|5|4|

**Strike Craft (Space)**
It will shoot down AA platforms or other strike craft
| **Cost** | **Upkeep** | **HP** | **Movement** | **Attack** | **Fuel Cost** | **Range** |
|----|----|----|----|----|----|----|
|5/5|1/5|1|1|1|5|2|

**Combat**
When it comes to ground combat the idea is that Infantry shut down artillery (if they are between the artillery and their target), Tank blows away infantry, Artillery destroys tanks, missiles can be shot shown with artillery. Missles shoot down space stations, AA platforms counter missiles, Strike craft counter AA platforms. Then when it comes to meteors crashing into stuff the only thing it should not 1 hit ko/analyze unites in the local area is the capitals, or maybe it could as they could build a stellar engine and get it back into orbit or redirect it, though a missile should not blow up a meteor.

**Remarks**
So the idea is that they have to manage two layers with the ground being an eternal battle and space starting off as an arms race due to the costly nature of it where they need to carefully plan out there strikes. If I had to simplify the game I would cut away the ground combat and allow the players to build ground anti-missile platforms and add another unite to combat asteroids that are made to land on the enemy.
